server/json/user1-institutions.json

```diff
@@ -6,47 +6,53 @@
       "status": {
         "code": 0,
         "message": ""
-      }
+      },
+      "currentSubmission": 0
```

Comment:
Do we need `currentSubmission`? Can't we use the `timestamp` in `submissions.json`?